frontend/public/storage/components/storage-overview/storage-overview.jsx

```diff
@@ -130,10 +156,18 @@ export class StorageOverview extends React.Component {
       fetchPrometheusQuery(CEPH_OSD_DOWN_QUERY, response => this.onFetch(PROM_RESULT_CONSTANTS.cephOsdDown, response));
       fetchPrometheusQuery(CEPH_OSD_UP_QUERY, response => this.onFetch(PROM_RESULT_CONSTANTS.cephOsdUp, response));
 
-      fetchPrometheusQuery(UTILIZATION_IOPS_QUERY, response => this.onFetch(PROM_RESULT_CONSTANTS.iopsUtilization, response));
-      fetchPrometheusQuery(UTILIZATION_LATENCY_QUERY, response => this.onFetch(PROM_RESULT_CONSTANTS.latencyUtilization, response));
-      fetchPrometheusQuery(UTILIZATION_THROUGHPUT_QUERY, response => this.onFetch(PROM_RESULT_CONSTANTS.throughputUtilization, response));
-      fetchPrometheusQuery(UTILIZATION_RECOVERY_RATE_QUERY, response => this.onFetch(PROM_RESULT_CONSTANTS.recoveryRateUtilization, response));
+      fetchPrometheusQuery(this._getUtilizationQuery('iops', '[60m:10m]'), response => this.onFetch(PROM_RESULT_CONSTANTS.iopsUtilizationForOneHr, response));
+      fetchPrometheusQuery(this._getUtilizationQuery('iops', '[360m:60m]'), response => this.onFetch(PROM_RESULT_CONSTANTS.iopsUtilizationForSixHr, response));
+      fetchPrometheusQuery(this._getUtilizationQuery('iops', '[1440m:240m]'), response => this.onFetch(PROM_RESULT_CONSTANTS.iopsUtilizationForTwentyFourHr, response));
+      fetchPrometheusQuery(this._getUtilizationQuery('latency', '[60m:10m]'), response => this.onFetch(PROM_RESULT_CONSTANTS.latencyUtilizationForOneHr, response));
+      fetchPrometheusQuery(this._getUtilizationQuery('latency', '[360m:60m]'), response => this.onFetch(PROM_RESULT_CONSTANTS.latencyUtilizationForSixHr, response));
+      fetchPrometheusQuery(this._getUtilizationQuery('latency', '[1440m:240m]'), response => this.onFetch(PROM_RESULT_CONSTANTS.latencyUtilizationForTwentyFourHr, response));
+      fetchPrometheusQuery(this._getUtilizationQuery('throughput', '[60m:10m]'), response => this.onFetch(PROM_RESULT_CONSTANTS.throughputUtilizationForOneHr, response));
```

Comment:
why are we having the same queries for overlapping durations? can't we make just one query instead?

Reply:
after looking at https://github.com/kubevirt/web-ui-components/pull/462. Maybe it would be the best to come up with a mechanism to register queries which the component needs.

Reply:
@suomiy Do you have any idea how can we register/call the queries from the web-ui-components? Can you provide some example?

Reply:
relevant effort: https://github.com/openshift/console/pull/1591/files. Maybe we should wait with this until storage is moved to openshift/console.

Not sure if we should commit to optimizing here. @rawagner thoughts?

Reply:
We can implement the logic for Utilization Card now and it will be reused with a few changes when moved to openshift console upstream.

Reply:
+1 makes sense